scripts/build

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const archiver = require('archiver');
+
+module.exports = (
+  buildZip('slate-src', ['src'], ['config-sample.yml'])
+);
+
+function buildZip(name, directories = [], files = []) {
+  const output = fs.createWriteStream(`upload/${name}.zip`);
```

Comment:
I got `Error: ENOENT: no such file or directory, open 'upload/slate-src.zip'` here... so I didn't have an `upload` directory. Once I added it, it was fine. So maybe check for that dir first? You can use `stat` to see if it's a directory or not.